docs/setup-options-overview.md

```diff
@@ -12,10 +12,6 @@ At a minimum, all of the options will transpile and serve the Grapher code, whic
 
 This needs a working MySQL database and gives you an interactive admin UI for visually editing our charts as well as all our ~4000 charts and their data.
 
-**Full WordPress setup** 🚀
```

Comment:
if we're keeping `make up.full`, I think we should be coherent and explain here what this setup is for. However, t this point, the only difference I could notice between `make up` and `make up.full` was related to the cloudflare functions dev server. It used to be that the overhead of the `full` variant justified the split and associated maintenance, but this is much less true now, and we should just merge them and have a single local dev command.